src/components/JobCard/JobCard.tsx

```diff
@@ -0,0 +1,79 @@
+import { FC } from 'react'
+import { Stack, Grid, Avatar, Typography, Button, Chip, Card, CardContent } from '@mui/material'
+import BusinessIcon from '@mui/icons-material/Business'
+import LocationOnIcon from '@mui/icons-material/LocationOn'
+import AccountBalanceWalletIcon from '@mui/icons-material/AccountBalanceWallet'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import JobData from '@/types/JobListing'
+
+
+
+interface JobCardProps {
+    job: JobData
+}
+
+const JobCard: FC<JobCardProps> = ({ job }) => {
+    const handleReadMoreClick = () => {
+        window.open(job.jobPostingUrl, '_blank', 'noopener,noreferrer')
+    }
+
+    return <Card sx={{ bgcolor: '#512da8', color: 'white', width: '100%' }}>
```

Comment:
As much as possible, let's reference colors from `theme/theme.ts` to avoid different hex codes on different pages. I removed `white` as I didn't notice it doing anything here.

```suggestion
    return <Card sx={{ bgcolor: theme.palette.primary.light, width: '100%' }}>
```

We'll need to import the theme at the top, and let's consider redefining `primary.light` to something like `purple[700]'. 

The built-in color gradients are a good starting point. https://mui.com/material-ui/customization/color/#color-palette

We are also not using the secondary or tertiary colors yet, so we can consider redefining those.